### PR TITLE
Dev challenge 69

### PR DESCRIPTION
**Pull request by gaebar**
create a new brunch for exercise 69

### CODE REVIEW

File: exercises/tuples-lists-dictionaries/challenge_069.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+country_names = ("England", "Italy", "Spain", "France", "Portugal")
+
+print(country_names)
+
+
+# Ask for user input and ensuring the user entering an integer
+def get_user_input(message):
+    while True:
+        user_input = input(message)
+        if user_input.isnumeric():
```
Comment (rrees) [inline exercises/tuples-lists-dictionaries/challenge_069.py]: Here I think you just need to check whether the input is in the tuple and loop until the user enters something that is present

File: exercises/tuples-lists-dictionaries/challenge_069.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+def print_msg(user_choice, country_name):
+    print(f"{user_choice} has index number {country_names.index(user_choice)}")
+
+
+def main():
+    user_choice = ask_for_user_choice()
+    print_msg(user_choice, user_choice)
+
+
+main()
```
Comment (rrees) [inline exercises/tuples-lists-dictionaries/challenge_069.py]: Have a look at this piece in the documentation on the [conventional way](https://docs.python.org/3.7/library/__main__.html) to define `main` in a Python script.

**Review by rrees — COMMENTED** [review]
A few minor things but this looks good :+1:

File: exercises/tuples-lists-dictionaries/challenge_069.py (hunk 1 above)
Comment (gaebar) [reply exercises/tuples-lists-dictionaries/challenge_069.py]: simplified this logic

File: exercises/tuples-lists-dictionaries/challenge_069.py (hunk 2 above)
Comment (gaebar) [reply exercises/tuples-lists-dictionaries/challenge_069.py]: thanks - I have amended the code to include __main__ handling

**Comment by rrees** [comment]
You can go ahead and merge now

File: exercises/tuples-lists-dictionaries/challenge_069.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+def ask_for_user_choice():
+    while True:
+        user_choice = get_user_input("Please enter one of the countries above: ")
+        if user_choice not in country_names:
+            print("This country is not in the list, please try again:")
+        return user_choice
+
+
+def print_msg(user_choice, country_name):
```
Comment (rrees) [inline exercises/tuples-lists-dictionaries/challenge_069.py]:
```suggestion
def print_message(user_choice, country_name):
```

Generally: stop abbreviating names it isn't helpful

**Review by rrees — APPROVED** [review]

**Comment by gaebar** [comment]
Resolved the challenge 69 - tuple.

**Pull request merged**